app/src/main/java/com/example/seminar1/SignUpActivity.kt

```diff
@@ -17,18 +17,18 @@ class SignUpActivity : AppCompatActivity() {
         binding = ActivitySignUpBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        ClickSignUpDone()
+        initAuthButtonClickListeners()
 
     }
     //회원가입 완료 버튼 클릭시
-    private fun ClickSignUpDone(){
+    private fun initAuthButtonClickListeners(){
         binding.btnSignupDone.setOnClickListener {
             val name = binding.etSignupName.text.toString()
             val id = binding.etSignupId.text.toString()
             val pw = binding.etSignupPw.text.toString()
 
             //name,id,pw 셋 중 하나라도 비어있다면
-            if(name.isEmpty() || id.isEmpty() || pw.isEmpty()){
+            if(name.isBlank() || id.isBlank() || pw.isBlank()){
```

Comment:
```suggestion
            if (listOf(name, id, pw).any(String::isBlank)) {
```
이런 재미있는 문법도 가능합니다. 굳이 쓸 필요는 없습니다. 더 좋은 코드인지도 모르겠고